README.md

```diff
@@ -1,10 +1,25 @@
 # Scalismo-ui
 
-Scalismo-ui is a library for visualizing statistical shape models and related 3D objects, such as 3D meshes or 3D medical images. It is designed to complement the [Scalismo](https://github.com/unibas-gravis/scalismo) framework and to make it easy to build simple, graphical shape modelling applications and to allow for "visual debugging" of complex image analysis algorithms.
+Scalismo-ui is a library for visualizing statistical shape models and related 3D objects, such as 3D meshes or 3D medical images. It is designed to complement the [Scalismo](https://github.com/unibas-gravis/scalismo) framework and to make it easy to build  graphical shape modelling applications and to allow for "visual debugging" of complex image analysis algorithms.
 Further, it can be used as a standalone application for visualizing shape models, 3D meshes and 3D medical images.
 
 Scalismo-ui was developed and is currently maintained by the [Graphics and Vision Research Group](http://gravis.cs.unibas.ch) of the [University of Basel](http://www.unibas.ch).
 
+### Why
+
+We believe that visualization is a key factor in the development of complex image analysis algorithm and that visual inspection is the most powerful debugging tool. But it is only used if visualization is simple to use. In Scalismo-ui, every object can be visualized by adding only one line of code:
+
+```scala
+val objectView = ui.show(object, "name of object")
```

Comment:
To be on the safe side, I would create a group here, to avoid that people try to show 2 models in the same group